indicators/NO_FUNC_001-004/R/NO_FUNC_001-004.qmd

```diff
@@ -0,0 +1,2825 @@
+pivot_longer(NO_FUNC_001_2022to2024[['high']], cols = 2:7, names_to = "region", values_to = "high")[,3],
+pivot_longer(NO_FUNC_001_2022to2024[['low']], cols = 2:7, names_to = "region", values_to = "low")[,3],
+pivot_longer(NO_FUNC_001_2022to2024[['n']], cols = 2:7, names_to = "region", values_to = "n")[,3]
+)
+)
+
+## naturally open lowland
+NO_FUNC_001a <- rbind(
+cbind(
+ecosystem="naturally_open_lowland",period="2019to2021",
+pivot_longer(NO_FUNC_001a_2019to2021[['median']], cols = 2:7, names_to = "region", values_to = "median"),
+pivot_longer(NO_FUNC_001a_2019to2021[['high']], cols = 2:7, names_to = "region", values_to = "high")[,3],
+pivot_longer(NO_FUNC_001a_2019to2021[['low']], cols = 2:7, names_to = "region", values_to = "low")[,3],
+pivot_longer(NO_FUNC_001a_2019to2021[['n']], cols = 2:7, names_to = "region", values_to = "n")[,3]
+),
+cbind(
+ecosystem="naturally_open_lowland",period="2022to2024",
+pivot_longer(NO_FUNC_001a_2022to2024[['median']], cols = 2:7, names_to = "region", values_to = "median"),
+pivot_longer(NO_FUNC_001a_2022to2024[['high']], cols = 2:7, names_to = "region", values_to = "high")[,3],
+pivot_longer(NO_FUNC_001a_2022to2024[['low']], cols = 2:7, names_to = "region", values_to = "low")[,3],
+pivot_longer(NO_FUNC_001a_2022to2024[['n']], cols = 2:7, names_to = "region", values_to = "n")[,3]
+)
+)
+
+## semi-natural
+NO_FUNC_001b <- rbind(
+cbind(
+ecosystem="semi_natural",period="2019to2021",
+pivot_longer(NO_FUNC_001b_2019to2021[['median']], cols = 2:7, names_to = "region", values_to = "median"),
+pivot_longer(NO_FUNC_001b_2019to2021[['high']], cols = 2:7, names_to = "region", values_to = "high")[,3],
+pivot_longer(NO_FUNC_001b_2019to2021[['low']], cols = 2:7, names_to = "region", values_to = "low")[,3],
+pivot_longer(NO_FUNC_001b_2019to2021[['n']], cols = 2:7, names_to = "region", values_to = "n")[,3]
+),
+cbind(
+ecosystem="semi_natural",period="2022to2024",
+pivot_longer(NO_FUNC_001b_2022to2024[['median']], cols = 2:7, names_to = "region", values_to = "median"),
+pivot_longer(NO_FUNC_001b_2022to2024[['high']], cols = 2:7, names_to = "region", values_to = "high")[,3],
+pivot_longer(NO_FUNC_001b_2022to2024[['low']], cols = 2:7, names_to = "region", values_to = "low")[,3],
+pivot_longer(NO_FUNC_001b_2022to2024[['n']], cols = 2:7, names_to = "region", values_to = "n")[,3]
+)
+)
+
+## all ecosystems
+NO_FUNC_001to004 <- rbind(
+  NO_FUNC_004,NO_FUNC_003,NO_FUNC_002,NO_FUNC_001a,NO_FUNC_001b
+)
+
+```
+
+
+
+
+## 10. Results
+Ecosystem indices (median), uncertainty (0.25 and 0.75 quantiles, i.e. 'low' and 'high'), and number of observations (n) for each region, period and indicator.
+
+Indicators:
+fpci.min = overall functional plant community indicator, based on the minimum of the underlying indicators
+CC, SS, RR =  Grime's C, S and R
+xxx1 = lower xxx indicator (i.e. deviations towards too little xxx)
+xxx2 = upper xxx indicator (i.e. deviations towards too much xxx)
+
+```{r}
+
+DT::datatable(NO_FUNC_001to004)
```

Comment:
The indicator column should contain the indicator ID's. These should be unique combinations of ecosystems and metric (different regions can have the same ID). So, there are many indicators here, more than four. See issue #240. This requires fixing before we can merge. In the issue I give two options.

Reply:
For some reason the datatable desn't print when I view the html using githack. But I can see it when I render locally. I think this is a web browser issue. Let's just ignore this for now.